spreadsheet/update/update_query.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+//
+// Use of this source code is governed by the terms of the Affero GNU General
+// Public License version 3.0 as published by the Free Software Foundation and
+// appearing in the file LICENSE included in the packaging of this file. A
+// commercial license can be purchased on https://unidoc.io.
+
+package update
```

Comment:
Add a package comment describing the package.   Something like
// package update contains definitions needed for updating columns.

Reply:
I think it only works if you add it right above with no empty lines above it

Reply:
Can you run golint on the package and make sure it is clean?